src/net/sourceforge/kolmafia/Modifiers.java

```diff
@@ -2724,14 +2804,15 @@ public void lookupFamiliarModifiers(
     switch (familiarId) {
       case FamiliarPool.HATRACK:
         if (famItem == EquipmentRequest.UNEQUIP) {
-          this.add(Modifiers.HATDROP, 50.0, "Familiar:naked hatrack");
-          this.add(Modifiers.FAMILIAR_WEIGHT_CAP, 1, "Familiar:naked hatrack");
+          this.addDouble(Modifiers.HATDROP, 50.0, ModifierType.FAMILIAR, "naked hatrack");
+          this.addDouble(Modifiers.FAMILIAR_WEIGHT_CAP, 1, ModifierType.FAMILIAR, "naked hatrack");
         }
         break;
       case FamiliarPool.SCARECROW:
         if (famItem == EquipmentRequest.UNEQUIP) {
-          this.add(Modifiers.PANTSDROP, 50.0, "Familiar:naked scarecrow");
-          this.add(Modifiers.FAMILIAR_WEIGHT_CAP, 1, "Familiar:naked scarecrow");
+          this.addDouble(Modifiers.PANTSDROP, 50.0, ModifierType.FAMILIAR, "naked scarecrow");
+          this.addDouble(
+              Modifiers.FAMILIAR_WEIGHT_CAP, 1, ModifierType.FAMILIAR, "naked " + "scarecrow");
```

Comment:
another split string

Reply:
thanks, this was IntelliJ being overenthusiastic.